examples/xdisplay_mirror.py

```diff
@@ -20,20 +20,35 @@
 
 import click
 import numpy as np
-from PIL import ImageGrab
+from PIL import Image, ImageGrab
 
 import adafruit_blinka_raspberry_pi5_piomatter as piomatter
 import adafruit_blinka_raspberry_pi5_piomatter.click as piomatter_click
 from adafruit_blinka_raspberry_pi5_piomatter.pixelmappers import simple_multilane_mapper
 
+RESAMPLE_MAP = {
+    "nearest": Image.NEAREST,
+    "bilinear": Image.BILINEAR,
+    "lanczos": Image.LANCZOS,
+    "bicubic": Image.BICUBIC
+}
+
 
 @click.command
 @click.option("--mirror-region", help="Region of X display to mirror. Comma seperated x,y,w,h. "
                                       "Default will mirror entire display.", default="")
 @click.option("--x-display", help="The X display to mirror. Default is :0", default=":0")
+@click.option("--resample-method",
+              help="The resample method for PIL to use when resizing the screen image."
+                   "Valid values are: nearest, bilinear, lanczos, and bicubic. Default is nearest",
+              default="nearest")
```

Comment:
Tip: you can use click.Choice to automatically handle the list of alternatives. (and delete the `if resample_method not in` check below)

```diff
 @click.option("--resample-method",
-              help="The resample method for PIL to use when resizing the screen image."
-                   "Valid values are: nearest, bilinear, lanczos, and bicubic. Default is nearest",
+              help="The resample method for PIL to use when resizing the screen image.",
+              type=click.Choice(RESAMPLE_MAP),
               default="nearest")
```

Reply:
Nifty, Thank you! that is updated in the latest commit to use click.Choice